pkgs/by-name/it/itk_4_13/package.nix

```diff
@@ -0,0 +1,145 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  expat,
+  fftw,
+  fftwFloat,
+  hdf5-cpp,
+  libjpeg,
+  libtiff,
+  libpng,
+  libuuid,
+  xz,
+  vtk,
+  zlib
+}:
+  # this ITK version is old and is only required for OTB package
```

Comment:
If this package is very specfic to otb and uses custom patches from them, we should either work with overwrites or move it local to that package

Reply:
Thanks @SuperSandro2000 !

I'll move it to local to that package.

Reply:
@SuperSandro2000 I have moved `itk_4_13` inside the `otb` folder.